<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-0KMM1L6H" failures="1" tests="1" name="runner.RunnerClass" time="106.976" errors="0" timestamp="2023-03-17T21:10:59 IST" skipped="0">
  <testcase classname="runner.RunnerClass" name="runScenario" time="106.976">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at pages.FlipkartPage.removeFromCart(FlipkartPage.java:84)
at stepDefn.StepDefnClass.remove_item_from_the_cart(StepDefnClass.java:82)
at âœ½.Remove Item from the Cart(file:///C:/Users/sutharsan/eclipse-workspace/CucumberWithTestNG/src/test/resources/features/sanity.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.RunnerClass -->
